src/chat/flagsReducers.js

```diff
@@ -0,0 +1,58 @@
+import {
```

Comment:
Because it will cause a message list repaint.
We probably want to normalize and domain-nize the data format anyways.
So the state.messages thing will not be identical to the getMessages() return data as it is now.
I discussed with Tim the possibility to get 'short' messages and 'detailed' messages.
The short ones would be, say IDs and flags, the long ones will be full content.
So we can get 1000 short messages on start etc.